back/src/workout/workout.service.ts

```diff
@@ -49,6 +59,10 @@ export class WorkoutService {
     return this.workoutRepository.findById(workoutId)
   }
 
+  async getByProgramId(programId: string): Promise<Workout[]> {
+    return this.workoutRepository.findByProgramId(programId)
+  }
+
```

Comment:
You can remove if not necessary.

If necessary, needs to be justified by a test.

Otherwise, it's like dead code.